### ISSUE TITLE
need to define rtoldefault for Dual

### ISSUE THREAD

**Issue by tpapp** [JuliaDiff/ForwardDiff.jl#176]
Eg
```
import Base: rtoldefault

rtoldefault{N, T <: Real}(::Type{ForwardDiff.Dual{N,T}}) = rtoldefault(T)
```
does the trick. Otherwise it falls back to `0`, and `isapprox` fails.

**Comment by tpapp** [comment]
PR #177.

**Comment by jrevels** [comment]
fixed by #177
